fmriprep/workflows/fieldmap/se_pair_workflow.py

```diff
@@ -101,41 +97,6 @@ def se_pair_workflow(name='Fieldmap_SEs', settings=None):  # pylint: disable=R09
 
     return workflow
 
-def fieldmap_to_phasediff(name='Fieldmap2Phasediff'):
```

Comment:
InterfacedWorkflows looks really nice--what are the blockers to getting it into nipype?